docs/integrations/app-integrations/youtube.mdx

```diff
@@ -5,66 +5,79 @@ sidebarTitle: YouTube
 
 In this section, we present how to connect YouTube to MindsDB.
 
-[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users can upload, view, share, and interact with videos created by individuals and organizations from around the world.
+[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users
+can upload, view, share, and interact with videos created by individuals and organizations from around the world.
 
-Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments, detect sentiment of comments, or use AI models to create replies.
+Comment Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments,
+detect sentiment of comments, or use AI models to create replies.
 
-## Connection
+## Install the Handler
```

Comment:
Throughout the app integration docs, we use sections "Connection" and "Usage".  Let's keep it consistent.
So let's change it back to Connection.